lib/administrate/field/nested_has_many.rb

```diff
@@ -27,7 +27,7 @@ class Engine < ::Rails::Engine
       DEFAULT_ATTRIBUTES = %i(id _destroy).freeze
 
       def nested_fields
-        associated_form.attributes.reject do |nested_field|
+        associated_form.attributes.values.flatten.reject do |nested_field|
```

Comment:
Do you have any thoughts on how we could make this backwards compatible?

I'm thinking that we could either check the version, but that might be bad, or reflect on the type of `attributes`.

Reply:
And, of course, any thoughts on how we might go about testing it?

Reply:
Maybe not the nicest, but something like 

```ruby
if Administrate::VERSION < "0.20.0"
  associated_form.attributes
else
  associated_form.attributes.values.flatten
end
```